client.js

```diff
@@ -43,8 +43,8 @@
     /**
      * 指定したチャットのメッセージを取得
      */
-    ChatWork.prototype.getMessages = function(room_id) {
-      return this.get('/rooms/' + room_id + '/messages');
+    ChatWork.prototype.getMessages = function(room_id, force = 0) {
+      return this.get('/rooms/' + room_id + '/messages?force' + force);
```

Comment:
```suggestion
      return this.get('/rooms/' + room_id + '/messages?force=' + force);
```

PRありがとうございます！`=` が必要そうかなと思いました 🙏

Reply:
@cw-shibuya 
申し訳ございません。完全なタイポでした😆 
修正しましたのでご確認ください。